src/py21cmfast/inputs.py

```diff
@@ -775,14 +798,18 @@ class AstroParams(StructWithDefaults):
         units. Default is `ION_Tvir_MIN`.
     F_H2_SHIELD: float, optional
         Self-shielding factor of molecular hydrogen when experiencing LW suppression.
-        Cf. Eq. 12 of Qin+2020
+        Cf. Eq. 12 of Qin+2020. Obsolete in new version.
```

Comment:
I see. Still, the new prescription includes shielding self-consistently from the sims, so the user should not play with that parameter unless they go back to Macachek+01, right?

> I also had to checkout the MINIHALO test files from the master branch. You might have altered this as well?

Yeah I tried to regenerate those with the new defaults, but my output must've been broken. Thanks for fixing those!

> Let me know if you want me to update the test files and use A_LW=2.0 and BETA_LW=0.6 instead.

I think we should change the defaults to be A_LW=2 and BETA_LW=0.6, since the Macachek values produce nearly a factor of 10 more feedback, so it's not very realistic.

Reply:
You mean they consider self-shielding of LW? I didn't realize that. I this case, we can probably remove F_H2_SHIELD, or simply emphasize here that users should only use this parameter for extra shielding... You could decide then

Reply:
I kept it with extra comments to the user. No reason to remove it entirely, I think, it may be useful for someone some day.